wire-ios/Wire-iOS/Sources/Authentication/Event Handlers/Flow Start/AuthenticationStartClientLimitErrorHandler.swift

```diff
@@ -17,16 +17,17 @@
 //
 
 import Foundation
+import WireNetwork
 
 final class AuthenticationStartClientLimitErrorHandler: AuthenticationEventHandler {
 
     weak var statusProvider: AuthenticationStatusProvider?
 
     func handleEvent(
         currentStep: AuthenticationFlowStep,
-        context: (NSError?, Int)
+        context: (BackendEnvironment2?, NSError?, Int)
```

Comment:
Here and elsewhere: these changes are because the context has an extra parameter that isn't used in most places. I'm not so concerned about the niceness of the legacy authentication code as will probably delete it soon.